ministryofjustice/correspondence_tool_staff
# Add previous BU IDs to BU reports

## Description
This PR adds two extra columns to the business unit reports to add the ids of each business unit, and the previous ID of the Unit if it has been moved.

## Self-review checklist


### Screenshots


### Related JIRA tickets
[CT-2822](https://dsdmoj.atlassian.net/browse/CT-2822)

### Deployment


### Manual testing instructions
1. if testing locally you might want to move some teams around so that the previous BU ids are populated.
2. down load some BU reports
3. should see two extra columns next to "Business unit" on the right: "Business unit ID" and "Previous business unit ID"
